### PR TITLE
Hasanein

### PR DESCRIPTION
**Pull request by hasaninqh**
The website has been created based on the requirments. It is also responsive to all devices viewport.

### CODE REVIEW

**Comment by LucyMac** [comment]
Hi @hasaninqh thanks for submitting your homework. 

What are all the Javascript files in there? Where do they come from?

Please remove them. You shouldn't need to add any files, everything you needed was provided for you: one index.html file, a css folder with 2 files, and an img folder. A Favicon and a font file. That's it. Please delete everything else. 

Once this is done I will review your work. 

Thanks

**Comment by hasaninqh** [comment]
Hi Lucy

These files came when i was trying to copy the twitter and fb icons .

I would try to clean the folder and resend it to you

File: css/style.css [hunk 1]
```diff
@@ -1,14 +1,182 @@
...
+}
+.alignment{
+    text-align: center;
+}
+
+.sec1{
+    background-image: url("../img/first-background.jpg");
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    padding-top: 20%;
```
Comment (LucyMac) [inline css/style.css]: the background image is being squashed on small screens. Try `background-size: cover` and background-position` if you need to move it around

File: css/style.css [hunk 2]
```diff
@@ -1,14 +1,182 @@
...
+        background-color: linen;
+        width:200px; 
+        display: block;
+        margin:3% 0 0 0; 
+        padding:5px;
+    }
+    .sec2{
+        
+        flex-direction: column;
+
```
Comment (LucyMac) [inline css/style.css]: `sec1` would look better with a smaller height on mobile.

File: index.html [hunk 3]
```diff
@@ -2,18 +2,91 @@
...
+        <div class="sec1"> 
+            <h1 class="text">
+                Introducing Karma
+            </h1>
+            <h2 class="text text2">
+                Bring WiFi with you, everywhere you go.  
+            </h2>
+            <a href="" class="btn">Learn more</a>
+        </div>
+        <h1 class="text">Every one needs a little Karam</h1>
```
Comment (LucyMac) [inline index.html]: this headline needs a little breathing space.

File: index.html [hunk 4]
```diff
@@ -2,18 +2,91 @@
...
+                
+                </div> 
+        </div>
+    </div>  
+</main>  
+     <!-- footer -->
+<footer>
+    <div class="footer alignment">
+            <div class="footer-content">
+
```
Comment (LucyMac) [inline index.html]: give a little breathing space to your footer

**Comment by hasaninqh** [comment]
The last pull request is for the new changes made based on your recommendation and also bootstrap was used to make the webpage responsive

**Pull request closed without merging**